<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Критерии поиска</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Выборка</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Слой</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mLayerComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Поле</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="mLogicComboBox">
        <item>
         <property name="text">
          <string>И</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ИЛИ</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Логика</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Способ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="mMethodComboBox">
        <item>
         <property name="text">
          <string>Включая</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Исключая</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="mUseRegistry">
        <property name="text">
         <string>Регистр</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="mWellName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Полное имя</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Имя скважины</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Номер по API</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Размещение</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Номер слота</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Оператор</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Долгота</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Широта</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="mLocation"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Автор</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="mOperator"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="mFullName"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="mApiNumber"/>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="mSlotNumber"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="mLatitude"/>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="mDateTime"/>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="mAuthor"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="mLongitude"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Дата/Время</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="mApiNumberToolButton">
        <property name="toolTip">
         <string>Получить список значений поля в выборке</string>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QToolButton" name="mLongitudeToolButton">
        <property name="toolTip">
         <string>Получить список значений поля в выборке</string>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="mWellsToolButton">
        <property name="toolTip">
         <string>Получить список значений поля в выборке</string>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QToolButton" name="mLocationToolButton">
        <property name="toolTip">
         <string>Получить список значений поля в выборке</string>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QToolButton" name="mDateToolButton">
        <property name="toolTip">
         <string>Получить список значений поля в выборке</string>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="mFullNameToolButton">
        <property name="toolTip">
         <string>Получить список значений поля в выборке</string>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QToolButton" name="mSlotNumberToolButton">
        <property name="toolTip">
         <string>Получить список значений поля в выборке</string>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QToolButton" name="mLatitudeToolButton">
        <property name="toolTip">
         <string>Получить список значений поля в выборке</string>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="mOperatorToolButton">
        <property name="toolTip">
         <string>Получить список значений поля в выборке</string>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QToolButton" name="mAuthorToolButton">
        <property name="toolTip">
         <string>Получить список значений поля в выборке</string>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
